bin/revdep-rebuild.sh

```diff
@@ -320,7 +321,8 @@ normalize_emerge_opts() {
 setup_color() {
 	# This should still work if NOCOLOR is set by the -C flag or in the user's
 	# environment.
-	[[ $NOCOLOR = yes || $NOCOLOR = true ]] && export RC_NOCOLOR=yes # HACK! (grr)
+	[[ -n $NO_COLOR || $NOCOLOR = yes || $NOCOLOR = true ]] &&
+		export RC_NOCOLOR=yes # HACK! (grr)
```

Comment:
Do wonder if this is actually needed given gentoo-functions supports it now, but eh. Let's not worry for now.

Reply:
The latest released version (0.19) doesn't support `NO_COLOR`.

Reply:
Actually, we'd still have to export one of `NO_COLOR` or `RC_NOCOLOR`, because gentoo-functions doesn't respect `NOCOLOR`.